zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/instance/DbElementInstanceState.java

```diff
@@ -292,6 +292,27 @@ public void deleteProcessInstanceKeyByDefinitionKey(
         processInstanceKeyByProcessDefinitionKey);
   }
 
+  @Override
+  public void decrementNumberOfTakenSequenceFlows(
```

Comment:
🔧 Let's move this method directly under the other `decrementNumberOfTakenSequenceFlows` so they stay close together.